### DISCUSSION TITLE
Is possible to delete a pending task that blocks all the others?

### DISCUSSION THREAD

**Discussion by Bakwla** [gotson/komga#1627] [Q&A] [answered]
### Steps to reproduce

1 Press "Cancel All Tasks" from server Management page
2 Server cancel all pending task except one: "AnalyzeBook: 1"
3 the server don't seems to execute other tasks added
4 I have modified the number of task threads from 1 to 4 and the server now seems to execute the added tasks but the first one is still pending and the yellow line don't stop running.

### Expected behavior

Stop all the tasks pressing "Cancel All Tasks" from server Management page

### Actual behavior

Under the logo the windows says 1 pending task AnalyzeBook: 1 
and the yellow line don't stop running

### Komga version

1.11.1

### Operating system

Synology NAS App: Container Manager

### Installation method

Docker

### Acknowledgements

- [X] I have searched the existing issues (open AND closed) and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/docs/faq).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Not sure to understand what this is about tbh.

**Comment by Bakwla** [comment]
Sorry if the description is lacking; please be patient this is my first post and the first problem I have with Komga.
In a nutshell there is always a "1 pending task AnalyzeBook: 1" and the situation does not change whether I restart the server or press "Cancel All Tasks" from Server Management page.
If the number of task threads in the Server Settings page is set to 1 the other tasks remain in the queue without being executed; if the number is greater the other tasks are executed but the first one (AnalyzeBook:1) continues to remain pending. 
I can't find a solution to cancel it

I turned off the server, renamed the log file, turned the server back on, and attach the new log file 
[komga.log](https://github.com/user-attachments/files/16329573/komga.log)

thank you in advance for helping

**Reply by gotson** [reply]
it seems the file cannot be analyzed properly, apparently it's a RAR5. You could try repackaging it in a zip (cbz).

**Comment by Bakwla** [comment]
Ok but how canI find what file it is?
I am unable to read the log file correctly and completely
Please could you point me to the line where the name of the file is so I can find and delete it?
tia

**Reply by gotson** [reply]
`file:/data/Fumetti d'epoca 2/Albo d'oro (poi Albi d'oro) v2 - v3 Serie della Prateria - v3 Serie Comica Collection (Mondadori 1946-1956)/Albo d'oro della prateria a01n33 - Pecos Bill 2a serie 51, Il tam-tam del voo-doo (Mondadori 1953-08-20) %5Bc2c leo066 dinofelix%5D 1.0.cbr`

**Comment by Bakwla** [comment, marked as answer]
Solved + today I learned something 
Thank you a lot gotson
